src/components/UI/KubernetesVersionLabel.tsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+interface IKubernetesVersionLabelProps {
+  version: string;
```

Comment:
Typechecking fails because this isn't marked as optional in the props interface, but the prop-types say that it is optional (`PropTypes.string.isRequired` would make it not optional).

When using default props, the easiest way is to just do

```suggestion
  version?: string;
```